cypress/e2e/functional/researcher_anonymous_classes.spec.js

```diff
@@ -0,0 +1,149 @@
+import * as c from '../../support/constants.js'
+import * as researchClassesHelper from "../../support/helpers/researcherClassesHelper.js"
+import {ResearcherClassesElements as researchClasses} from "../../support/elements/researcher_anonymous_classes_elements";
+
+context("Researcher classes side bar, filters, resutls table", () => {
+
+  before(function() {
+    cy.visit(c.LEARN_PORTAL_BASE_URL); // Visit LEARN Portal home page
+    cy.login(c.TEACHER3_USERNAME, c.TEACHER3_PASSWORD); // Login as teacher user
+  });
+
+  after(function() {
+    cy.logout();
+    cy.clearAllCookies();
+  });
+
+  it("Verify research project in sidebar", () => {
+    researchClassesHelper.expandResearchProjects(); 
+    researchClassesHelper.getResearchProjectsOpen().should("exist");
+  });
+  it("Verify research classes landing page", () => {
```

Comment:
Given that the "it" statements are what we are limited by, I think these should be combined into one "it", with multiple "log" statements where the "it" statements are now.

Reply:
Done